tasks/request_files/request_files.py

```diff
@@ -361,6 +347,39 @@ def inner_task(
         "asyncOperationId": event[EVENT_CONFIG_KEY][CONFIG_JOB_ID_KEY],
     }
 
+def get_glacier_recovery_type(config: Dict[str, Any]) -> str:
```

Comment:
Remember to pull this new function out of existing unit tests to maintain unit test practices. You should be able to delete any new `task` tests you added, and simplify existing `task` tests.

Reply:
updated unit tests.